example/pubspec.yaml

```diff
@@ -16,7 +16,6 @@ publish_to: "none" # Remove this line if you wish to publish to pub.dev
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 version: 1.0.0+1
```

Comment:
I think publish_to should be none for the example since it won't be published anywhere?

Reply:
For an example app, it should not really matter... Especially, as you cannot publish whole *apps* on pub.dev (aka Dart projects with an entrypoint importing flutter) anyway.